src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -180,26 +169,18 @@ public bool UsingDefaultValue
             {
                 usingDefaultValue = value;
                 RaisePropertyChanged("UsingDefaultValue");
-                RaisePropertyChanged("ToolTipContent");
+                RaisePropertyChanged("ToolTip");
             }
         }
 
         /// <summary>
-        /// Controls whether the Use Default Value option is available.
-        /// </summary>
-        [JsonIgnore]
-        public bool DefaultValueEnabled
```

Comment:
I've got one concern here and this is a lot of zt node authors desire to actually use null as a default value...  maybe that design decision is out of scope for this PR and this won't actually change the behavior (I dont think it will) just wanted to mention it.